static/sass/styles.scss

```diff
@@ -341,3 +341,9 @@ h2 {
     text-decoration: none;
   }
 }
+
+// XXX Ant - 31.05.19: This can be removed when the follow issue is resolved.
+// https://github.com/canonical-web-and-design/vanilla-framework/issues/2364
```

Comment:
These types of comments are always in here years later when someone stumbles across them :-D

Can you create a tracking issue pleaseeeeee

Reply:
A tracking issue is likely to fall into the same problem. This method has worked well on other sites as it gives a clear indication to whether the snippet can be removed. I would rather keep the same system.